editor/src/components/canvas/canvas-strategies/strategies/set-padding-strategy.spec.browser2.tsx

```diff
@@ -21,7 +21,88 @@ import {
 } from '../../ui-jsx.test-utils'
 import { SetPaddingStrategyName } from './set-padding-strategy'
 
+const edgePieces: Array<EdgePiece> = ['top', 'bottom', 'left', 'right']
```

Comment:
I _think_ we have a convention to make these global consts capitalized, ie EdgePieces

Reply:
thanks, updated!

Reply:
thanks, updated!